core/src/main/scala/org/http4s/MimeDB.scala

```diff
@@ -0,0 +1,6700 @@
+package org.http4s
+
+private[http4s] object MimeDB {
```

Comment:
`MimeDB` is checked in rather than auto generated during the build. This means builds are stable but we could be stale with respect to updates on the upstream db.
There are pros/cons about the alternatives:

## Generated on build:
* Pros:
1. Always up to date
2. We don't need to check in this large file

* Cons:
1. Difficult to make the generator using http4s/circe without producing conflicts on the build
2. The build could break due to upstream changes
3. Cannot build it offline (Though in my experiments the downloaded file was cached)

## Checked in:
* Pros:
1. Stable builds and we can safely refer to `MimeDB`
2. This large file would totally kill a `Scala.js` version. If we check this in we can have a `MimeDB` for `scala.js` which is essentially empty

* Cons:
1. `MimeDB` could get stale
2. Eventually somebody could edit this

Reply:
I dislike checking in generated code, but I'm a big fan of repeatable builds.  A compromise is a generator pinned to a git hash.  But that complicates the build, and makes keeping it up to date a chore, so maybe it's the worst of both worlds.

I agree with your pros and cons and don't have a strong opinion.  Would love to hear from others here.

Reply:
is this every build, or triggered by a command?

Reply:
At the moment it is just created with a separate utility and manually added to `Http4s` repo and formatted